backend/prisma/schema.prisma

```diff
@@ -19,11 +19,17 @@ model Board {
   title    String
   category String
   author   String
+  imageURL String 
   cards    Card[]
 }
 
 model Card {
-  id      Int   @id @default(autoincrement())
-  boardId Int
-  board   Board @relation(fields: [boardId], references: [id])
+  id          Int   @id @default(autoincrement())
+  author      String 
+  imageURL    String 
+  upvotes     Int 
+  boardId     Int
```

Comment:
though there is no standard case for abbreviations, i've often seen people capitalize 2 characters and camel case for more characters

boardId -> boardID
imageURL -> imageUrl

(ref: https://stackoverflow.com/a/15526526)

I'm ok you use whichever standard.